test/e2e/scheduling/predicates.go

```diff
@@ -936,7 +940,12 @@ func WaitForSchedulerAfterAction(f *framework.Framework, action Action, ns, podN
 	}
 	success, err := observeEventAfterAction(f.ClientSet, f.Namespace.Name, predicate, action)
 	framework.ExpectNoError(err)
-	framework.ExpectEqual(success, true)
+	if expectSuccess && !success {
+		framework.Failf("event was failed after performing the supplied action for pod %v", podName)
+	}
+	if !expectSuccess && success {
```

Comment:
```suggestion
	if !expectSuccess && !success {
```

That's why I say that the name of the variable `success` is confusing. Can you rename it to `observed`?

Reply:
Sure!

Reply:
Done!